src/App.js

```diff
@@ -1,25 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Form from "./components/Form/Form.js";
+import { useState } from "react";
+import { uid } from "uid";
 
-function App() {
+export default function App() {
+  const [activities, setActivity] = useState([
+    { name: "swim", isForGoodWeather: true, id: "1234" },
+  ]);
+
+  function handleAddActivity(newActivity) {
+    console.log(newActivity);
+    setActivity([...activities, { ...newActivity, id: uid() }]);
+  }
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <header></header>
+      <Form text="hallo sandeep" onAddActivity={handleAddActivity} />
```

Comment:
I assume, you used the prop text only for testing. Please remove it.
```suggestion
      <Form onAddActivity={handleAddActivity} />
```